package.json

```diff
@@ -23,7 +23,7 @@
     "@spotify/eslint-config-oss": "^1.0.0",
```

Comment:
blah? ☝️

Reply:
sry about that haha :) snuck in there during some debugging